tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -0,0 +1,146 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "DrawingTest.h"
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <ImageIO/ImageIO.h>
+
+#include <Starboard/SmartTypes.h>
+#include <memory>
+
+static const CGSize g_defaultCanvasSize{ 512.f, 256.f };
+
+woc::unique_cf<CGColorSpaceRef> testing::DrawTest::s_deviceColorSpace;
+
+void testing::DrawTest::SetUpTestCase() {
+    s_deviceColorSpace.reset(CGColorSpaceCreateDeviceRGB());
+}
+
+void testing::DrawTest::TearDownTestCase() {
+    s_deviceColorSpace.release();
+}
+
+CGSize testing::DrawTest::CanvasSize() {
+    return g_defaultCanvasSize;
+}
+
+void testing::DrawTest::SetUp() {
+    CGSize size = CanvasSize();
+
+    _context.reset(CGBitmapContextCreate(
+        nullptr, size.width, size.height, 8, size.width * 4, s_deviceColorSpace.get(), kCGImageAlphaPremultipliedFirst));
```

Comment:
>size.width * 4 [](start = 45, length = 14)

1) are we ever going to have tests that render to colorspaces with not 4 components?
2) if not, call we call that out in a comment?